src/docs/development/tools/devtools/memory.md

```diff
@@ -38,77 +47,100 @@ in granularity:
+
+<dl markdown="1">
+<dt markdown="1">**Pause**</dt>
+<dd>Pause the memory overview chart to allow inspecting
+    the currently plotted data. Incoming memory data is still received;
+    notice the Range selector continues to grow to the right.</dd>
+<dt markdown="1">**Resume**</dt>
+<dd>The memory overview chart is live and displaying the
+    current time and the latest memory data received.</dd>
+<dt markdown="1">**Clear**</dt>
+<dd>All collected data plotted in charts and event timeline are deleted and all snapshots, analyses and allocation monitoring is deleted too.</dd>
+<dt markdown="1">**Display dropdown**</dt>
 <dd>All memory data collected (timeseries).
-    The left-most or first time/data (memory information),
-    in the selector is when the application was launched.
-    The right-most or last time/data is the continual memory
-    information being received (live) until the application is
-    stopped.</dd>
-<dt markdown="1">**Range selector view**</dt>
-<dd>Detailed view of the data collected
-    for this timeseries range (non-gray area).</dd>
-<dt markdown="1">**X-axis timestamp**</dt>
-<dd>Time of the collected memory information
-    (capacity, used, external, RSS (resident set size),
-    and GC (garbage collection).</dd>
-<dt markdown="1">**Hover information of collected data**</dt>
-<dd>At a particular time (x-axis) the detailed collected
-    memory data.</dd>
-<dt markdown="1">**Garbage collection occurred**</dt>
-<dd>Compaction of the heap occurred.</dd>
-<dt markdown="1">**Event timeline**</dt>
-<dd>When a user action occurred (such as the
-    Snapshot or Reset button being clicked).</dd>
+    The left-most or first time/data (memory information) when the application was launched. The right-most or last time/data is the continual memory information received (live) until the application is stopped.  The timeline is scaled by the selected "Display" dropdown e.g., 
+<dt markdown="1">- Display 1 Minute</dt>
+<dt markdown="1">- Display 5 Minutes</dt>
+<dt markdown="1">- Display 10 Minutes</dt>
+<dt markdown="1">- Display All Minutes (slider disabled)</</dt>
+</dd>
+<dt markdown="1">**Source**</dt>
+<dd>Source can be either "Live Feed" current connected running Flutter application or offline "Export" file.</dd>
+<dt markdown="1">**Android Memory**</dt>
+<dd>Displays or hides the Android Memory Chart.</dd>
+<dt markdown="1">**GC**</dt>
+<dd>Initiates a garbage collection - compaction of the heap</dd>
+<dt markdown="1">**Export**</dt>
+<dd>Saves collected data for Event Timeline, Memory Overview Chart and Android Overview Chart. Files saved are displayed under the Source drop-down. Selecting a file will load the offline data.</dd>
+</dl>
+
+## Memory Actions
+
+Below the memory charts (Event Timeline, Memory Overview and Android Overview charts) are actions to collect information about memory e.g., Snapshots, Table Tree view, Treemap view, Groupings, or Monitor Allocations, Searching and Filtering.
```

Comment:
"are actions to collect and display information about memory usage." 
I think we can remove everything after e.g. since they are described below

Reply:
Done.